extensions/amp-nexxtv-player/0.1/test/test-amp-nexxtv-player.js

```diff
@@ -33,54 +34,49 @@ describes.realWin(
       doc = win.document;
     });
 
-    async function getNexxtv(attributes, opt_responsive) {
-      const nexxtv = doc.createElement('amp-nexxtv-player');
-
-      for (const key in attributes) {
-        nexxtv.setAttribute(key, attributes[key]);
-      }
-      nexxtv.setAttribute('width', '111');
-      nexxtv.setAttribute('height', '222');
-      if (opt_responsive) {
-        nexxtv.setAttribute('layout', 'responsive');
-      }
```

Comment:
I removed this `responsive` option since it can be set as an attribute anyway. This is an ugly pattern that I'm trying to remove from video player tests in general.

Similarly, I removed a test that ensured that the element had its responsive classname applied. This was unnecessary since it tests a runtime feature.